piyushgarg-dev/review-app
# fix: When pencil icon in form edit page is clicked the focus is on the form name and user can edit the form name

## What does this PR do?

This PR basically fixes the bug that is when the pencil icon in the form edit page (beside form name) is pressed the focus is turned on on the form name and the user can edit the form name.

Fixes #142 

BEFORE

https://github.com/piyushgarg-dev/review-app/assets/131521505/a0329a9d-dc06-4167-b105-a77f47545192

AFTER

https://github.com/piyushgarg-dev/review-app/assets/131521505/360e2d60-3521-4933-acf6-020999a63bf5

## Type of change


## How should this be tested?
1. Create a project.
2. Create a form.
3. Click on edit icon.
4. Now click on the pencil icon in the form edit page.

If you have an existing project and a form just click on http://app.localhost:3000/dashboard/subdomain/forms/ and click on edit icon to redirect to form edit page.

- Are there environment variables that should be set?
- No

- What are the minimal test data to have?
- Project, Form

## Mandatory Tasks
